core/services/relay/evm/contract_transmitter.go

```diff
@@ -157,6 +163,13 @@ func (oc *ContractTransmitter) LatestConfigDigestAndEpoch(ctx context.Context) (
 }
 
 // FromAccount returns the account from which the transmitter invokes the contract
-func (oc *ContractTransmitter) FromAccount() ocrtypes.Account {
+func (oc *contractTransmitter) FromAccount() ocrtypes.Account {
 	return ocrtypes.Account(oc.transmitter.FromAddress().String())
 }
+
+func (oc *contractTransmitter) Start(ctx context.Context) error { return nil }
+func (oc *contractTransmitter) Close() error                    { return nil }
+
+// Has no state/lifecycle so it's always healthy and ready
+func (oc *contractTransmitter) Healthy() error { return nil }
+func (oc *contractTransmitter) Ready() error   { return nil }
```

Comment:
Hmmm is there some reason that it must implement this interface?

Reply:
Same question from me, do we really need to implement the interface that requires these methods? I would not like to have these noop functionality that's empty just to comply to an interface.

Reply:
Yeah, because ContractTransmitter is now a `services.Service`. The mercury transmitter implements starting/stopping/healthy/ready, this one is stateless so they are noops.